index.html

```diff
@@ -97,14 +122,14 @@ <h1>Multi-Post
                     <button>javascript</button>
                 </ul>
                 <button class="see_project">See project</button>
-
+            </div>
             </div>
         </section>
     </main>
     <!-- footer -->
     <footer>
-        <section class="section1">
-            <h1>About Myself</h1>
+        <div class="section1">
+            <h3>About Myself</h3>
```

Comment:
You've done a great job with your project! **I have a couple of suggestions to further improve it:**

- [ ] **[OPTIONAL]** It would be beneficial to move your about section out of the `footer` tag and use a `section` element instead. This change **aligns with best practices and improves the structure of your `HTML`.** 🎯

- [ ] I noticed that **your About section is centered**, but according to the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122), the elements should fill the entire container. As a hint, **the left element should take up 40% of the page's width, while the right element should occupy the remaining 60%**. Adjusting the layout in this way **will bring it closer to the desired design.** 😃

   ### Your design:
   ![image](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/107952724/a5fb6a3f-90cd-4c41-8350-029cdbe24e48)
   
   ### Figma design:
   ![image](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/107952724/73937fce-ef09-41ae-ad88-0751761743b8)

Reply:
resolved